README.md

```diff
@@ -16,47 +16,59 @@ Frontend Service can be found [here](`https://github.com/Emergency-Response-Demo
 
   You can deploy using Dockerfile on OpenShift Webconsole or run the following command.
   
-  `oc new-app --name find-service --strategy docker https://github.com/Emergency-Response-Demo/find-service`
+  ```
+  oc new-app --name find-service --strategy docker https://github.com/Emergency-Response-Demo/find-service
+  ```
   
   You will need to pass `INCIDENT_SERVICE_URL` and `MISSION_SERVICE_URL` as Environment variable.
   
 ## Deploying using Tekton Pipeline
 
 * Prerequisite
 
-    OpenShift v4.x Cluster. You can install using `OpenShift Pipelines Operator` from the Operator Hub.
+    - OpenShift v4.x Cluster.
+     
+    - Tekton Pipeline Installed. You can install using `OpenShift Pipelines Operator` from the Operator Hub.
     
-    You will also use the Tekton CLI (tkn). Download the Tekton CLI by following [instructions](https://github.com/tektoncd/cli#installing-tkn) available on the CLI GitHub repository.
+    - Tekton CLI (tkn) (Optional). Download the Tekton CLI by following [instructions](https://github.com/tektoncd/cli#installing-tkn) available on the CLI GitHub repository.
 
 * Installation Steps
 
     1. Fork this repository so that you can create a trigger for any GitHub Event which will trigger the pipeline and deploy the new code.
     
     2. Clone the repository to your local machine.
     
-        ` git clone https://github.com/<your-github-username>/find-service`
+        ```
+        git clone https://github.com/<your-github-username>/find-service
+       ```
     
     3. Traverse to the pipeline folder.
     
-        ` cd find-service/pipeline/`
+        ```
+        cd find-service/pipeline/
+        ```
         
     4. Login to your OpenShift Cluster.
```

Comment:
Would it be better to specify this in the prerequisites itself?
Something like: `OpenShift 4v.x Cluster. Logged in with developer permissions.`?